Index: qgis-3.28.3/external/untwine/untwine/VoxelKey.hpp
===================================================================
--- qgis-3.28.3.orig/external/untwine/untwine/VoxelKey.hpp
+++ qgis-3.28.3/external/untwine/untwine/VoxelKey.hpp
@@ -12,6 +12,7 @@
 
 #include <string>
 #include <algorithm>
+#include <cstdint>
 
 #pragma once
 
Index: qgis-3.28.3/external/lazperf/Extractor.hpp
===================================================================
--- qgis-3.28.3.orig/external/lazperf/Extractor.hpp
+++ qgis-3.28.3/external/lazperf/Extractor.hpp
@@ -36,6 +36,7 @@
 
 #include <cstring>
 #include <vector>
+#include <cstdint>
 
 #include "portable_endian.hpp"
 
Index: qgis-3.28.3/external/untwine/api/QgisUntwine.hpp
===================================================================
--- qgis-3.28.3.orig/external/untwine/api/QgisUntwine.hpp
+++ qgis-3.28.3/external/untwine/api/QgisUntwine.hpp
@@ -3,6 +3,7 @@
 #include <map>
 #include <string>
 #include <vector>
+#include <cstdint>
 
 #ifdef _WIN32
 #include <Windows.h>
