Index: ultrajson-5.4.0/deps/double-conversion/double-conversion/utils.h
===================================================================
--- ultrajson-5.4.0.orig/deps/double-conversion/double-conversion/utils.h
+++ ultrajson-5.4.0/deps/double-conversion/double-conversion/utils.h
@@ -75,6 +75,7 @@ inline void abort_noreturn() { abort();
     defined(_POWER) || defined(_ARCH_PPC) || defined(_ARCH_PPC64) || \
     defined(__sparc__) || defined(__sparc) || defined(__s390__) || \
     defined(__SH4__) || defined(__alpha__) || \
+    defined(__loongarch64) || \
     defined(_MIPS_ARCH_MIPS32R2) || \
     defined(__AARCH64EL__) || defined(__aarch64__) || defined(_M_ARM64) || \
     defined(__riscv)
